src/code42cli/cmds/legal_hold.py

```diff
@@ -238,6 +263,50 @@ def _print_matter_members(username_list, member_type="active"):
         echo("No {} matter members.\n".format(member_type))
 
 
+def _merge_matter_members_with_devices(sdk, user_dataframe):
+    devices_generator = sdk.devices.get_all(active="true", include_backup_usage=True)
+    device_list = _get_total_archive_bytes_per_device(devices_generator)
+    devices_dataframe = DataFrame.from_records(
+        device_list,
+        columns=[
+            "userUid",
+            "guid",
+            "name",
+            "osHostname",
+            "status",
+            "alertStates",
+            "orgId",
+            "lastConnected",
+            "version",
+            "archiveBytes",
+        ],
+    )
+    return user_dataframe.merge(
+        devices_dataframe, how="inner", on="userUid"
+    ).reset_index(drop=True)
+
+
+def _build_user_dataframe(users):
+    user_dataframe = DataFrame.from_records(
+        users, columns=["activeMembership", "userUid", "username"]
+    )
+    return user_dataframe
+
+
+def _get_total_archive_bytes_per_device(devices_generator):
+    device_list = [device for page in devices_generator for device in page["computers"]]
+    for i in device_list:
+        archive_bytes = [archive["archiveBytes"] for archive in i["backupUsage"]]
+        i["archiveBytes"] = sum(archive_bytes)
+    return device_list
+
+
+def _print_storage_by_org(devices_dataframe):
```

Comment:
This method does seem to actually print any storage by org

Reply:
I'm not sure I understand the comment. Do you mean _doesn't_ actually print storage by org? If so, you're right. It just _gets_ it.

Reply:
The name of the method suggests it prints it

Reply:
I miswrote the original comment